Braintree/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,7 @@
 
     <queries>
         <package android:name="com.venmo" />
+        <package android:name="com.paypal.android.p2pmobile" />
```

Comment:
Why are we adding this? I didn't think we ever switched to the PayPal app anymore.

Reply:
Yeah Preferred Payments still checks if PayPal is installed similar to Venmo, so we went ahead and added them in.

Reply:
Oh, nice! Makes sense.